changelogs/fragments/67454-ovirt_disk-correct-description-of-storage_domain.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - "ovirt_disk: correct description of storage_domain, there is no default value of the atribute"
```

Comment:
```suggestion
  - "ovirt_disk: correct description of storage_domain, there is no default value of the attribute"
```

Reply:
@mnecas you marked this conversation as resolved but you didn't accept the suggestion - @Akasurde is correct about the spelling of `attribute` - can you accept the suggestion?